dashboard/test/controllers/api_controller_test.rb

```diff
@@ -1288,16 +1288,6 @@ def make_progress_in_section(script)
     )
   end
 
-  test 'should show teacher-dashboard link when a teacher' do
-    teacher = create :teacher
-    sign_in teacher
-
-    get :user_menu
-
-    assert_response :success
-    assert_select 'a[href="//test.code.org/teacher-dashboard"]', 'Teacher Home Page'
-  end
-
```

Comment:
Since we don't want there to be a link in the dropdown that goes to the Teacher Dashboard with the navigation update, we don't need a test asserting that it's there. 

Just deleting a failing test feels wrong somehow; do I need to write a follow up test that ensures the Teacher Homepage link is NOT in the dropdown once the changes are no longer cookie-dependent?